src/Model/Config/Source/SearchType.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tweakwise\TweakwiseJs\Model\Config\Source;
+
+use Magento\Framework\Data\OptionSourceInterface;
+use Tweakwise\TweakwiseJs\Model\Enum\SearchType as SearchTypeEnum;
+
+class SearchType implements OptionSourceInterface
+{
+    /**
+     * @return array
+     */
+    public function toOptionArray(): array
+    {
+        return array_map(function (SearchTypeEnum $searchTypeEnum) {
+            return ['value' => $searchTypeEnum->value, 'label' => $searchTypeEnum->label()];
```

Comment:
Wellicht nog even label wrappen in `__()`, zodat er ook vertalingen toegevoegd kunnen worden.